### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by bishboria**
- Google credentials required refreshing
- Remove absolute path to configuration

Recreating https://github.com/alphagov/inventory-tool/pull/1 with better commits

### CODE REVIEW

**Comment by bishboria** [comment]
This work is on hold

File: app/services/google_credentials_generator.rb [hunk 1]
```diff
@@ -67,7 +66,7 @@ def substitute_env_vars
     @client_secret = data['client_secret']
     data['client_secret'] = nil
 
-    fp = File.open('/Users/stephen/src/gds/inventory-tool/config/new-config.json', 'w')
+    fp = File.open('config/google_drive_config.json', 'w')
```
Comment (malcolmbaig) [inline app/services/google_credentials_generator.rb]: hate to be picky but can we change `fp` to something more descriptive - `config_file` maybe?

Comment (bishboria) [inline app/services/google_credentials_generator.rb]: Yep, sorry. I'm bad for that.

File: app/services/google_credentials_generator.rb [hunk 2]
```diff
@@ -67,9 +66,9 @@ def substitute_env_vars
     @client_secret = data['client_secret']
     data['client_secret'] = nil
 
-    fp = File.open('/Users/stephen/src/gds/inventory-tool/config/new-config.json', 'w')
-    fp.write(JSON.pretty_generate(data))
-    fp.close
+    config_file = File.open('config/google_drive_config.json', 'w')
```
Comment (benlovell) [inline app/services/google_credentials_generator.rb]: If you use the block form of `File.open` and write to the yielded argument you get a `close` guarantee, for free. :sparkler:

**Pull request merged**